apps/partners/src/components/ProfilePopover.tsx

```
@@ -0,0 +1,100 @@
+import { useAppState } from "../state/appState"
+import * as React from "react"
+
+function getInitials(name?: string) {
+  if (!name) return "?"
+  const parts = name.split(" ")
+  if (parts.length === 1) return parts[0][0]?.toUpperCase() || "?"
+  return (parts[0][0] + parts[1][0]).toUpperCase()
+}
+
+export default function ProfilePopover({ session, organizations, currentOrg }: {
+  session: any,
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)